src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js

```diff
@@ -53,14 +54,22 @@ const QuickMarcEditorRows = ({
   const showDeleteConfirmation = useCallback(({ target }) => {
     setRemoveIndex(+target.dataset.index);
     toggleRemoveModal();
+    setFocusedRowIndex(+target.dataset.index);
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [setRemoveIndex]);
 
   const confirmDeletion = useCallback(() => {
     deleteRecord({ index: removeIndex });
     toggleRemoveModal();
+    setFocusedRowIndex(undefined);
```

Comment:
IMHO, undefined shouldn't be used as a value, there is a null to show that no value. undefined means more that it was never defined

Reply:
any actions with it?